sympy/geometry/point.py

```diff
@@ -784,6 +784,56 @@ def taxicab_distance(self, p):
         s, p = Point._normalize_dimension(self, Point(p))
         return Add(*(abs(a - b) for a, b in zip(s, p)))
 
+    def canberra_distance(self, p):
+        """The Canberra Distance from self to point p.
+
+        Returns the weighted sum of horizontal and vertical distances to
+        point p.
+
+        Parameters
+        ==========
+
+        p : Point
+
+        Returns
+        =======
+
+        canberra_distance : The weighted sum of horizontal and vertical
```

Comment:
It would be good here to mention how the distances are weighted since someone reading the docstring may not be reading the code.